assets/js/modules/analytics-4/datastore/properties.js

```diff
@@ -162,10 +162,12 @@ const fetchGetGoogleTagSettingsStore = createFetchStore( {
 
 // Actions
 const WAIT_FOR_PROPERTIES = 'WAIT_FOR_PROPERTIES';
+const MATCHING_ACCOUNT_PROPERTY = 'MATCHING_ACCOUNT_PROPERTY';
 
 const baseInitialState = {
 	properties: {},
 	propertiesByID: {},
+	matchingAccountProperty: false,
```

Comment:
Since this is a temporal value, let's rename it to make that a bit more apparent.
```suggestion
	isMatchingAccountProperty: false,
```

Reply:
Renamed.